src/System.Memory/tests/ReadOnlyMemory/Strings.cs

```diff
@@ -90,7 +90,12 @@ public static void AsMemory_TryGetArray_ReturnsFalse()
             Assert.False(MemoryMarshal.TryGetArray(m, out ArraySegment<char> array));
             Assert.Null(array.Array);
             Assert.Equal(0, array.Offset);
+#pragma warning disable xUnit2013 // Do not use Assert.Equal() to check collection size
```

Comment:
💡 Consider filing a pair of xUnit bugs for this:

1. The analyzer should not report this situation in current versions xUnit
2. The `Assert.Empty` implementation should be updated to special-case `ArraySegment<T>`

Reply:
Will do.

Reply:
https://github.com/xunit/xunit/issues/1970